app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt

```diff
@@ -99,12 +99,12 @@ class AddProfileActivityTest {
       )
       onView(withId(R.id.add_profile_activity_create_button)).perform(scrollTo())
       onView(withId(R.id.add_profile_activity_create_button)).perform(click())
-      intended(hasComponent(ProfileActivity::class.java.name))
+      intended(hasComponent(ProfileChooserActivity::class.java.name))
     }
   }
 
   @Test
-  fun testAddProfileActivity_changeConfiguration_inputName_clickCreate_checkOpensProfileActivity() {
+  fun testAddProfileActivity_changeConfiguration_inputName_clickCreate_checkOpensProfileChooserActivity() {
```

Comment:
If char in lines are more than 100 than we get a ktlint issue. To fix this we can do either thing: 
1. Reformat the name of the test with few abbreviations 
2. Disable ktlint check for that particular line of code by putting `// ktlint-disable max-line-length` at the end of the line. Example is available in the [wiki section](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#how-to-fix-the-most-common-issues)

Reply:
Also to see when your line of code is growing more than 100, you can have a grey line in Android Studio in editor area by simply going to `Editor-> Code Style->general->hard wrap at **100**`

Reply:
@anandwana001 Now for which method I should go? Actually I spotted at some place this comment line but I thought I am not allowed to use it.

Reply:
I think as of now you can go with second option, simply disabling the check. 
Also to be sure, run ktlint check on your machine before commit.